src/open_inwoner/templates/pages/sitemap/sitemap.html

```diff
@@ -0,0 +1,37 @@
+{% extends "master.html" %}
+{% load i18n list_tags %}
+
```

Comment:
For now this looks pretty good for styling.
I wonder is there is an easy way to "crawl" the OIP site so you can generate your own sitemap locally easily and then add the missing pages to this template.
We still need links to 'home' and the login-page + registration page and maybe I am forgetting some that are publicly visible, like the flat pages. So yeah: even if they already are in the footer, they need to be in the Sitemap as well.